qiskit_experiments/library/randomized_benchmarking/data/generate_clifford_data.py

```diff
@@ -109,7 +109,27 @@ def gen_clifford_compose_2q_gate():
         for rhs in _CLIFF_SINGLE_GATE_MAP_2Q.values():
             composed = cliff_lhs.compose(_CLIFF_2Q[rhs])
             products[lhs, rhs] = _TO_INT_2Q[_hash_cliff(composed)]
-    return products.tocsr()
+    return products.tocsc()
+
+
+def gen_clifford_compose_2q_dense() -> tuple[np.typing.NDArray[int], list[int]]:
+    """Generate a dense multiplication table for 1-qubit Clifford numbers
+
+    The multiplication table is generated from the sparse table generated by :meth:`gen_clifford_compose_2q_gate`.
```

Comment:
Maybe better to mention to the spec of the matrix returned, e.g. column=full Cliffords, row=subset of Cliffords (defined in _CLIFF_SINGLE_GATE_MAP_2Q) instead.